.github/workflows/tests.yml

```diff
@@ -214,7 +214,7 @@ jobs:
           pip install torch==$TORCH_VERSION
           pip install -r requirements.txt
           pip install -e .
-          pytest -k "test_all_allowlisted" --cov-fail-under 0
+          pytest -m slow --cov-fail-under 0
```

Comment:
nit but should we rename the job if we don't perform only allowlist tests anymore here?